cfme/tests/webui/test_splitter.py

```diff
@@ -66,7 +63,7 @@ def test_pull_splitter_persistence(request, appliance, model_object, destination
     navigate_to(appliance.server, 'Dashboard')
     try:
         navigate_to(model_object, destination)
-    except (TypeError, CannotScrollException):
+    except (TypeError, CannotScrollException, Exception):
```

Comment:
sometimes it rises `Exception: Could not open accordion VMs & Templates`

Reply:
**REQUIRED**
After adding `Exception` other exceptions don't make sense. You should be more specific. If the required exception doesn't exist, create your own and catch it here.